service_files/active-bmc-only.conf

```diff
@@ -0,0 +1,4 @@
+[Unit]
+ConditionPathExists=/run/openbmc/active-bmc
+After=phosphor-set-bmc-active-passive.service
+PartOf=obmc-bmc-active.target
```

Comment:
The contents are based on: https://gerrit.openbmc.org/c/openbmc/docs/+/70233/9/designs/redundant-bmc.md#244

Although we could change it, like remove the "After=" line since this service doesn't exist yet.

Reply:
I'm good if Matt is

Reply:
Talked with Matt. Adding the commit that adds the new service.